src/javafundamentals/arrays/Ex3AgeStatistics.java

```diff
@@ -6,4 +6,49 @@
  */
 
 public class Ex3AgeStatistics {
+    public static void main(String[] args) {
+        int[] ages = {20, 30, 40, 50, 60};
+        System.out.println(sum(ages));
+        System.out.println(average(sum(ages), ages));
+        System.out.println(youngest(ages));
+        System.out.println(oldest(ages));
+
+    }
+    //sum of ages method
+    public static double sum(int[] myNumber){
+        double answer = 0;
+        for(int x = 0; x<5; x++){
+            answer = answer + myNumber[x];
+        }return answer;
+    }
+    //average method
+    public static double average(double answer, int[] myNumber){
+        double averageAnswer = answer/myNumber.length;
+        return averageAnswer;
+    }
+
+    //youngest age
+    public static int youngest(int[] myNumber){
+        int youngest = 200;
```

Comment:
There is a constant that has the maximum value for an Integer, you can use it and forget about selecting a suitable maximum number:
`Integer.MAX_VALUE`